### PR TITLE
Wpsolr integration

### PR DESCRIPTION
**Pull request by eostis**
### Why:

Submission of WPSOLR as a new integration

### What's being changed:

integrations.yml and WPSOLR's logo.

### Type of change:

- [x] Documentation updates (non-breaking change which updates documents)

### How Has This Been Tested?

Just documentation changes.

### CODE REVIEW

**Comment by eostis** [comment]
Pull request to add WPSOLR as an integration

File: _data/integrations.yml [hunk 1]
```diff
@@ -19,4 +19,9 @@
   description: DocArray (from Jina AI) is a library for nested, unstructured data in transit, including text, image, audio, video, 3D mesh, etc. It allows deep-learning engineers to efficiently process, embed, search, recommend, store, and transfer the data with a Pythonic API. Weaviate can be used in combination with DocArray to store large amounts of vectorized data.
   image: /img/jina_logo.svg
   type: neural search framework
+- name: WordPress & WooCommerce search plugin from WPSOLR
+  website: https://www.wpsolr.com/feature-weaviate/
+  description: Power all or parts of your search with Weaviate. We've integrated all Weaviate modules, including transformers, Question & Answers, and CLIP to extend OpenSearch, Algolia, Solr, Solr Cloud, or Elasticsearch with vectors where it is best qualified.
```
Comment (bobvanluijt) [inline _data/integrations.yml:24]: @eostis would be good to just focus on the Weaviate integration. For the rest this looks ok.

**Review by bobvanluijt — CHANGES_REQUESTED** [review]
See below

**Comment by eostis** [comment]
I agree with the CLA https://www.semi.technology/playbooks/misc/contributor-license-agreement.html

**Review by bobvanluijt — APPROVED** [review]

**Comment by eostis** [comment]
I's live now. Thanks. 👍

**Comment by bobvanluijt** [comment]
Thank you for the contrib 😊👍

**Pull request merged**